OSSystems/meta-browser
# chromium: update to 87.0.4280.88

Stable release announcements since our latest update:
https://chromereleases.googleblog.com/2020/12/stable-channel-update-for-desktop.html

Test-built with:

* chromium-x11
  * dunfell, clang, MACHINE="intel-core2-32"
  * dunfell, clang, MACHINE="intel-corei7-64"
  * dunfell, clang, MACHINE="raspberrypi3"
  * gatesgarth, MACHINE="raspberrypi3"

* chromium-ozone-wayland:
  * dunfell, clang, MACHINE="intel-corei7-64"
  * dunfell, clang, MACHINE="qemuarm64"
  * gatesgarth, clang, MACHINE="raspberrypi3"
  * dunfell, clang, MACHINE="m3ulcb"

License changes
---------------
Added licenses:
  * third_party/fusejs/LICENSE
  - Tab Search: Add Fuse.js to third_party/
  - https://chromium-review.googlesource.com/c/chromium/src/+/2375728

  * third_party/hyphenation-patterns/LICENSE
  - Add third_party/hyphenation-patterns
  - https://chromium-review.googlesource.com/c/chromium/src/+/2254498

  * third_party/shaka-player/LICENSE
  - Add shaka player to //third_party
  - https://chromium-review.googlesource.com/c/chromium/src/+/2359572

Removed licenses:
  * third_party/openvr/src/LICENSE
  - Remove OpenVR runtime and references
  - https://chromium-review.googlesource.com/c/chromium/src/+/2415308

Android-only:
Added licenses:
  * third_party/android_ndk/NOTICE
  - Upgrade to NDK r20
  - https://chromium-review.googlesource.com/c/android_ndk/+/1754382

Removed licenses:
  * third_party/android_deps/libs/androidx_legacy_legacy_support_v13/LICENSE
  - Reland: "Remove unused androidx-legacy libraries"
  - https://chromium-review.googlesource.com/c/chromium/src/+/2405998

  * third_party/android_deps/libs/com_android_support_animated_vector_drawable/LICENSE
    third_party/android_deps/libs/com_android_support_appcompat_v7/LICENSE
    third_party/android_deps/libs/com_android_support_cardview_v7/LICENSE
    third_party/android_deps/libs/com_android_support_design/LICENSE
    third_party/android_deps/libs/com_android_support_gridlayout_v7/LICENSE
    third_party/android_deps/libs/com_android_support_leanback_v17/LICENSE
    third_party/android_deps/libs/com_android_support_mediarouter_v7/LICENSE
    third_party/android_deps/libs/com_android_support_palette_v7/LICENSE
    third_party/android_deps/libs/com_android_support_preference_leanback_v17/LICENSE
    third_party/android_deps/libs/com_android_support_preference_v14/LICENSE
    third_party/android_deps/libs/com_android_support_preference_v7/LICENSE
    third_party/android_deps/libs/com_android_support_recyclerview_v7/LICENSE
    third_party/android_deps/libs/com_android_support_support_v13/LICENSE
    third_party/android_deps/libs/com_android_support_support_vector_drawable/LICENSE
    third_party/android_deps/libs/com_android_support_transition/LICENSE
  - Remove unused com_android_support targets in //third_party/android_deps
  - https://chromium-review.googlesource.com/c/chromium/src/+/2425010

Fixes #429
